config/settings.yml

```diff
@@ -2,15 +2,31 @@ plugins:
   zoom_enabled:
     default: false
     client: true
-  zoom_api_key:
```

Comment:
Maybe this is all okay, but I'm worried about existing sites currently using this plugin if they upgrade. I know some things are deprecated, but does the plugin currently work as is with the zoom_api_key or zoom_verification_token? If so, maybe we don't make a breaking change and allow the old way to still work along with the new way and provide an upgrade path.

Reply:
This will need to be changed since we are moving from a JWT apps to SDK apps, so in reality nothing should work rn 🙈. By making the move they will need to get new keys althogether